docs/02-app/01-building-your-application/08-testing/02-jest.mdx

```diff
@@ -232,7 +234,7 @@ setupFilesAfterEnv: ['<rootDir>/jest.setup.ts']
 setupFilesAfterEnv: ['<rootDir>/jest.setup.js']
 ```
 
-Then, inside `jest.setup.ts`, add the following import:
+Then, inside `jest.setup.ts|js`, add the following import:
```

Comment:
```suggestion
Then, inside `jest.setup`, add the following import:
```
lets just drop the extension or pick one, having a `|` here might cause more confusion

Reply:
Sure, I've fixed it too.